### DISCUSSION TITLE
Install Security Onion 2.3.181 on HCI sangfor

### DISCUSSION THREAD

**Discussion by RedOsprey** [Security-Onion-Solutions/securityonion#9074] [Unsupported Versions] [answered]
Dear Security Onion Team. ask for your help and input, I plan to use Security Onion to do security analysis. and I have also downloaded the Security Onion 2.3.181.iso file, when I try to install it on the HCI VM I get an error.
thing I want to ask.
1. to install do I have to install ubuntu/centos first on the VM and then install Security Onion 2.3.181.iso or directly I run Security Onion 2.3.181.iso on the HCI VM

best regards

**Comment by dougburks** [comment, marked as answer]
If you're using our Security Onion 2.3.181 ISO image, then you should be able to install it directly in the VM. For more information, please see https://docs.securityonion.net/en/2.3/installation.html. If you're a first time user, then you might want to start with a simple Import installation as shown at https://docs.securityonion.net/en/2.3/first-time-users.html.

**Reply by RedOsprey** [reply]
thank you for your support,
now I have successfully installed Security Onion 2.3.181 ISO image. yesterday there was an error because it was wrong when configuring the IP address, currently Security Onion is on the network, and the web base is already accessible.
how do I know Security Onion is working properly, for the network scheme I can learn where, please advice
![image](https://user-images.githubusercontent.com/114890934/200472478-d748f5d0-dff6-4173-92a8-f68773575a44.png)

**Reply by dougburks** [reply]
> how do I know Security Onion is working properly, for the network scheme I can learn where, please advice

We have lots of documentation at:
https://securityonion.net/docs

Also, check out our YouTube videos at:
https://securityonion.net/youtube

Finally, take a look at our free Security Onion Essentials training:
https://securityonion.net/training

**Reply by RedOsprey** [reply]
thanks for your support
